pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
+const MinHostPort int = 0
+const MaxPort int = 65536
+
+const intervalValueErrorMsg string = "must be %d or more and less than %d"
```

Comment:
word-smithing for feel

"must be greater than %d and less than %d" or "must be between %d and %d, inclusive"